tests/sps/validation/test_aff.py

```diff
@@ -1347,442 +1347,31 @@ def test_validate(self):
 
     def test_validate_affiliation_sub_article_original_only(self):
         self.maxDiff = None
-        xml_tree = xml_utils.get_xml_tree("tests/samples/1518-8787-rsp-56-79.xml")
+        xml = """
+                <article article-type="research-article" xml:lang="pt">
+                    <front>
+                        <article-meta>
+                            
+                        </article-meta>
+                    </front>
+                    <sub-article article-type="translation" id="TRpt" xml:lang="pt">
+                        <front-stub>
+                            <aff id="aff1002">
+                                <label>I</label>
+                                <country country="BR">Brasil</country>
+                                <institution content-type="original">Universidade Federal de Pelotas. Faculdade de
+                                    Medicina. Programa de Pós-Graduação em Epidemiologia. Pelotas, RS,
+                                    Brasil</institution>
```

Comment:
@Rossi-Luciano arrume o espaçamento do conteúdo de institution para os testes ficarem limpos.

Só use espaços em excessos se o teste for para considerar esta condição. Caso contrário só insere ruídos.

Reply:
OK @robertatakenaka